### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by schiessle**
Updated README.md to reflect the current state and to invite people to join the development

cc @Ivansss

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,7 @@
...
 
-This project is based on a [fork](https://github.com/strukturag/webrtc-ios) of Google's [WebRTC](https://chromium.googlesource.com/external/webrtc) library.
+This is a fork of the Spreed.me iOS app which was developed for the initial Spreed.me service. We will use this code as a basis to develop the iOS app for the Nextcloud Spreed video & voice conferencing app.
 
-You can use Spreed.ME iOS app with:
-- [Spreed.ME service](https://www.spreed.me)
-- [Spreedbox](https://www.spreed.me/spreedbox/)
-- Spreed.ME app for [Nextcloud](https://apps.nextcloud.com/apps/spreedme)/[ownCloud](https://apps.owncloud.com/content/show.php/Spreed.ME?content=174436)
-- Your own Spreed.ME [service](https://github.com/strukturag/spreed-webrtc)
+The app doesn't work yet with the Nextcloud Spreed app. But we invite everyone interested in it to join us in makeing the this app compatible with Nextcloud.
```
Comment (oparoz) [inline README.md]: typo: making

Comment (schiessle) [inline README.md]: thanks, fixed 😄

Comment (oparoz) [inline README.md]: You missed one, but my comment wasn't very clear ;)

s/makeing/making/

File: README.md [hunk 2]
```diff
@@ -1,38 +1,7 @@
-# Spreed.ME iOS app
-Spreed.ME is a messaging app that lets you securely send and receive messages and files and even start encrypted video and audiocalls - one-on-one or as conference.
+# Nextclod iOS app for the Spreed video & voice calls app
 
-This project is based on a [fork](https://github.com/strukturag/webrtc-ios) of Google's [WebRTC](https://chromium.googlesource.com/external/webrtc) library.
+This is a fork of the [Spreed.me iOS app](https://github.com/strukturag/spreedme-iosapp) which was developed for the initial Spreed.me service. We will use this code as a basis to develop the iOS app for the Nextcloud Spreed video & voice conferencing app.
```
Comment (nickvergessen) [inline README.md]: `Spreed.ME` *

😉

**Pull request merged**